bindings/python/src/create_torrent.cpp

```diff
@@ -129,10 +129,24 @@ namespace
         add_files(fs, file, [&](std::string const& i) { return cb(i); }, flags);
     }
 
-    void add_file(file_storage& fs, std::string const& file, std::int64_t size
+    void add_file0(file_storage& fs, string_view const& file, std::int64_t size
```

Comment:
the `string_view` should be passed by value, since it already acts as a pointer. (it should also be made `const`)